PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -255,7 +255,7 @@ Int_t AliEmcalJetTask::FindJets()
       if (fApplyArtificialTrackingEfficiency) {
         if (fTrackEfficiencyOnlyForEmbedding == kFALSE || (fTrackEfficiencyOnlyForEmbedding == kTRUE && tracks->GetIsEmbedding())) {
           
-          Double_t trackEfficiency = fTrackEfficiencyFunction.Eval(it->first.Pt());
+          Double_t trackEfficiency = 1.;//fTrackEfficiencyFunction.Eval(it->first.Pt());
```

Comment:
Can you please set this to `fTrackEfficiency` instead of 1? That way, we can continue to use the previous behavior until James can provide a fix.

Reply:
Sure - no problem! I will rebase before.

Cheers

Markus

Reply:
Thanks!